induyenugu

Induyenugu opened the pull request.

danawen commented:
Hi Indumathi, I checked your Grade Book assignment. I took off points because of a couple of small issues:
- The program needs to prompt the user to input student names and grades (5 pts)
- The program should store the students and the associated grades (as a comma-seperated string) in a Map<String,String> (5 pts)
If you fix those issues and would like a re-grade, let me know. Thanks!

danawen commented:
@Induyenugu I graded your Tic Tac Toe, Statistics, and Grade Book assignments.
- For Tic Tac Toe, I took off 5 pts because the game does not inform the user in the event of a tie/draw.
- For Grade Book, I took off 5 pts because the average is calculated incorrectly (I put in grades 90,92,94,96,98 and the program returned an average of 19.6)
Unfortunately, I'm unable to offer any more re-grades for these. Sorry about that.
Thanks!

Induyenugu commented:
@danawen thank you Dana for reviewing. I updated TicTacToe ame to verify Draw and Fixed Grades project. Grades is missing braces while doing average calculation which is giving wrong values. I sincerely request you if you can look into them for one last time.
Also I uploaded CarLot and Rentable. Can you please review them too.

Appreciate your time

danawen commented:
> @danawen thank you Dana for reviewing. I updated TicTacToe ame to verify Draw and Fixed Grades project. Grades is missing braces while doing average calculation which is giving wrong values. I sincerely request you if you can look into them for one last time.
> Also I uploaded CarLot and Rentable. Can you please review them too.
> 
> Appreciate your time

@Induyenugu - Regraded - thanks!

Induyenugu commented:
@danawen  Thank you very much Dana for your time. Much appreciated

danawen commented:
@Induyenugu For your Word Guess Game, I took off 5 pts because the program doesn't keep track of the incorrect letters guessed and inform the user (step 6 in the requirements list). It should say something like "Incorrect letters guessed so far: t,q,v,x" (for example)

Induyenugu commented:
@danawen, I don't see that in step 6. But any way I can show it and let you know once updated

Induyenugu commented:
@danawen , I updated word guess to show the wrong attempts count and wrong letters also. Please review

danawen commented:
Thanks @Induyenugu ! Just checked your Word Guess changes and updated your grade. Sorry for the delay.
